tests/robustness/traffic/etcd.go

```diff
@@ -73,6 +73,10 @@ type etcdTraffic struct {
 	largePutSize int
 }
 
+func (t etcdTraffic) ExpectUniqueRevision() bool {
```

Comment:
I'm curious about the reasons for putting these bools into functions versus just having it as part of traffic config? Are we expecting to have complex criteria in future for determining if a given traffic type will expect unique revisions? It feels like something that is more static to me but I might be missing something.

Reply:
Because it's property of traffic code, it could be inferred from write types used, however it's also not clear for them. For example there are three etcd requests types currently that generate multiple operation per revision.